nameko/testing/pytest.py

```diff
@@ -287,7 +287,9 @@ def socket_creator():
             '127.0.0.1', web_config_port)
         gr = eventlet.spawn(ws_app.run_forever)
         active_sockets.append((gr, ws_app))
-        return wait_for_sock()
+        socket = wait_for_sock()
+        socket.app = ws_app
```

Comment:
pass it in? not sure i follow. also, as i mentioned in the commit message, don't like the api but it maintains backwards compat. 

called it `socket` since it's the return value from `socket_creator`